server/models/User.py

```diff
@@ -0,0 +1,83 @@
+from flask.ext.bcrypt import Bcrypt
+from sqlalchemy import exc
+
+# (to do later when database is ready)
+# from database.py import db
+
+bcrypt = Bcrypt()
+
+
+class User(db.Model):
+  """User Model"""
+
+  __tablename__ = 'users'
+
+  id = db.Column(
+      db.Integer,
+      primary_key=True,
+  )
+
+  name = db.Column(
+      db.Text,
+      nullable=False,
+  )
+
+  email = db.Column(
+      db.Text,
+      nullable=False,
+      unique=True,
+  )
+
+  password = db.Column(
+      db.Text,
+      nullable=False,
+  )
+
+  def generate_user_object(self):
```

Comment:
I feel like to_dict or a method name like that is more appropriate

Reply:
noted